app/src/main/java/com/example/todoapp/view/fragment/CreateTodoFragment.kt

```diff
@@ -0,0 +1,51 @@
+package com.example.todoapp.view.fragment
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
+import com.example.todoapp.databinding.FragmentCreateTodoBinding
+import com.example.todoapp.model.Task
+import com.example.todoapp.viewmodel.AppViewModelFactory
+import com.example.todoapp.viewmodel.CreateTaskViewModel
+
+class CreateTodoFragment : Fragment() {
+
+    private lateinit var binding: FragmentCreateTodoBinding
+    private val viewModel: CreateTaskViewModel by lazy {
+        AppViewModelFactory(requireContext()).create(CreateTaskViewModel::class.java)
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        binding = FragmentCreateTodoBinding.inflate(layoutInflater)
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        initView()
+    }
+
+    private fun initView() {
+        binding.confirmButton.setOnClickListener {
```

Comment:
các sự kiện click a prefer impelement ở Fragment cho tập trung 
```suggestion
        binding.confirmButton.setOnClickListener(this)
```